src/Appwrite/Utopia/Response/Model/AttributeFloat.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model\Attribute;
+
+class AttributeFloat extends Attribute
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('format', [
```

Comment:
Same comment about normalization. Since 0.9 we also try to stay as close as possible to primitives (and successfully so far 😉). Instead of `format` attribute let's add `min`, and `max`. Attribute type can stay `float`. We can adjust the structure in the action after saving to the DB.